### DISCUSSION TITLE
Any way to change the UTC date to local time for created and updated field

### DISCUSSION THREAD

**Discussion by nikhil-varma-mudunuri** [pocketbase/pocketbase#3418] [Q&A] [answered]
same as title

**Comment by ganigeorgiev** [comment]
No, the `date` field values are stored and returned only in UTC.

**Reply by nikhil-varma-mudunuri** [reply]
is there any way I can view (admin UI) the date in local time (even though they are stored in UTC)?

**Reply by ganigeorgiev** [reply, marked as answer]
In earlier versions there was an option to show the local date on hover, but it was removed due to performance issues with the listing.

With the upcoming v0.19.0 the listing was refactored and optimized so I'll reconsider introducing it again.

**Comment by MehediHasannOvi** [comment]
i'm using v0.22.7 but i can't change UTC date to local time any way I can view (admin UI) the date in local time (even though they are stored in UTC)?

**Reply by ganigeorgiev** [reply]
Dates are always stored as UTC in the database.
The above mentioned local time representation is only for the Admin UI.
